docs/adapting-existing.md

```diff
@@ -1,5 +1,5 @@
 ---
-nav_order: 6
```

Comment:
This is a pain, I wonder should we add a 0 to all the nav_orders, like "70" in this case, that way it's easier to stick pages in between previous ones.

Reply:
Yep SGTM